py3status/modules/taskwarrior.py

```diff
@@ -29,6 +31,7 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 5
     format = '{task}'
+    task_args = 'start.before:today status:pending'
```

Comment:
Are you trying to use just `status:pending` in your new `task_args`?

Reply:
No, I use `+ACTIVE` (a virtual tag that tracks the tasks started using `tastk start`), so that the plugin always display the task I am working on